src/views/AllSourcesFeeds/SourceCard/index.tsx

```diff
@@ -27,6 +27,7 @@ function SourceCard(props: Props) {
         <Link
             className={styles.sourceCard}
             to={data?.url}
+            target="_blank"
```

Comment:
Let's add `external` prop and remove this target prop.

Reply:
resolving for now because the link is actual internal